### ISSUE TITLE
Support solr 7.0

### ISSUE THREAD

**Issue by jcoyne** [projectblacklight/blacklight#1786]
I believe this is because `solrQueryParser/@defaultOperator` is no longer supported: https://github.com/projectblacklight/blacklight/blob/4b634a6d40e043ba0dbcf9631350bb0765596d51/solr/conf/schema.xml#L551

More info:
https://issues.apache.org/jira/browse/SOLR-2724

**Comment by anarchivist** [comment]
```
Starting Solr...                                                                                                      solr-7.0.0.zip: |=============================================================================| 100% (Time: 00:00:15 )
solr-7.0.0.zip.md5: |=========================================================================| 100% (Time: 00:00:00 )
done.
rake aborted!
Failed to execute solr create:
Copying configuration to new core instance directory:
/var/folders/_8/kn1pqh5s7wzfvtjtxr5jtwlm0000gq/T/solr-7.0.0/server/solr/blacklight-core

Creating new core 'blacklight-core' using command:
http://localhost:8983/solr/admin/cores?action=CREATE&name=blacklight-core&instanceDir=blacklight-core


ERROR: Error CREATEing SolrCore 'blacklight-core': Unable to create core [blacklight-core] Caused by: Setting defaultSearchField in schema not supported since Solr 7
```

**Comment by cbeer** [comment]
Fixed by #1787
